kube-controllers/cmd/wrapper/wrapper.go

```diff
@@ -15,7 +15,7 @@
 package main
 
 import (
-	"github.com/projectcalico/calico/typha/pkg/cmdwrapper"
+	"github.com/projectcalico/calico/kube-controllers/pkg/cmdwrapper"
```

Comment:
I was actually thinking this might make sense as a generic binary / package that isn't tied to any one project. We use it in kube-ctrl in OSS, and Typha in enterprise - seems like a generic and well-scoped tool without being tied to any particular project?